xml/System/TimeOnly.xml

```diff
@@ -751,7 +759,15 @@ If `start` and `end` are equal, this method returns `false`, meaning there is ze
       <Docs>
         <summary>Represents the smallest possible value of <see cref="T:System.TimeOnly" />.</summary>
         <value>A <see cref="T:System.TimeOnly" /> that represents its smallest possible value.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+For <see cref="T:System.Numerics.IFloatingPoint`1> this method matches the IEEE 754:2019 `minimum` function. This requires NaN inputs to be propagated back to the caller and for `-0.0` to be treated as less than `+0.0`.
```

Comment:
```suggestion
For <xref:System.Numerics.IFloatingPoint%601>, this method matches the IEEE 754:2019 `minimum` function. This requires NaN inputs to be propagated back to the caller and for `-0.0` to be treated as less than `+0.0`.
```

Reply:
This looks like a bug, the description is wrong and shouldn't be about min/max functions

Reply:
I had fixed this for .NET 7 Preview 5 already.

Reply:
The old xmls got completely deleted. The new ref assemblies contained these types in System.Numerics so the xmls got created from scratch. What I am porting comes directly from triple slash.

Reply:
We should fix this again manually for P4 but it should be automatically correct for P5.